src/main/java/org/kainos/ea/model/JobRoleRequest.java

```diff
@@ -0,0 +1,80 @@
+package org.kainos.ea.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class JobRoleRequest {
```

Comment:
It looks like this is the model that we are using as the response. Typically these would be called Response Object. A better name for this class might be

```suggestion
public class JobRoleResponse {
```